successgilli/politico
# Ft setup postgres 163681596

#### What does this PR do?
It ensures that data sent to the database is persisted
#### Description of Task to be completed?
All endpoints previously created should be persisted with a database
#### How should this be manually tested?
after cloning the repo, cd into it, run 'npm i' and type npm run test.
The endpoint can also be tested using Postman or your browser
send data to either of the routes;
POST /api/v1/parties/
POST/api/v1/offices/
restart the server and navigate to either
GET /api/v1/parties/
GET/api/v1/offices/
the information sent should be available in an array of objects
#### Any background context you want to provide?
Not yet
#### What are the relevant pivotal tracker stories?
#163681596
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/42284402/52289475-6807e500-296e-11e9-980a-879972a28bb4.png)

#### Questions:
none
